src/main/java/org/datadog/jmxfetch/reporter/StatsdReporter.java

```diff
@@ -35,13 +35,18 @@ public StatsdReporter(String statsdHost, int statsdPort) {
 
     private void init() {
         initializationTime = System.currentTimeMillis();
+        /* Create the StatsDClient with "entity-id" set to "none" to avoid
+           having dogstatsd server adding origin tags, when the connection is
+           done with UDS. */
         statsDClient =
                 new NonBlockingStatsDClient(
                         null,
                         this.statsdHost,
                         this.statsdPort,
+                        Integer.MAX_VALUE,
```

Comment:
reference note: `Integer.MAX_VALUE` is the default queue length, so this will *NOT* change any behavior.